feat: KBOMG-261 - vl-alert-closed event

wimdetroyer opened the pull request.

kspeltix committed:
feat: UIG-2582 - storybook build in release-and-publish

wimdetroyer committed:
feat: KBOMG-261 - vl-alert-closed event

ksvlaanderen commented on libs/common/utilities/src/lib/stories/stories.helper.ts:
```diff
@@ -2,6 +2,19 @@ import { nothing } from 'lit';
+
+// Logt het event in zowel in Storybook als in de developer console.
+// Logt het event in de developer console omdat in Storybook custom event properties soms niet gelogd worden.
+export const logStorybookEvent = <T extends Event>(eventName: string) => {
```
We hebben in een openstaande PR een common-storybook module gemaakt, we zullen eerst die inmergen en dan dit verhuizen naar daar.
Door deze methode in de common-utilities te plaatsen komt storybook mee in de bundel wat niet de bedoeling is.

wimdetroyer replied:
Top, merci !

wimdetroyer replied:
Dus afhankelijk van: https://github.com/milieuinfo/uigov-web-components/pull/116

wimdetroyer replied:
De build lijkt hierdoor dus ook te falen: https://www.milieuinfo.be/bamboo/browse/UIGOV-CUWC66-1

Goldflow committed:
feat: UIG-2589 - vl-map-vector-layer - pas als layer ingesteld, styles instellen

Vroeger werd er niet gewacht tot de layer opgezet was, om styles in te stellen. Nu gebeurt dit wel.

Goldflow commented on libs/components/src/lib/alert/stories/vl-alert.stories-arg.ts:
```diff
@@ -1,11 +1,15 @@
 import { ALERT_ICON, ALERT_SIZE, ALERT_TYPE } from '../vl-alert.model';
+import { CATEGORIES } from '@domg-wc/common-utilities';
+import { VlAlertClosedEvent } from '../VlAlertClosedEvent';
```
we gaan typisch de files ook allemaal lower case houden; dus hier zou ik bv. eerder als filename `vl-alert-closed-event.model.ts` nemen

Goldflow commented on libs/components/src/lib/alert/vl-alert.component.ts:
```diff
@@ -1,6 +1,7 @@
 import { BaseElementOfType, webComponent } from '@domg-wc/common-utilities';
 import { alertStyle, iconStyle } from '@domg/govflanders-style/component';
 import { accessibilityStyle, resetStyle } from '@domg/govflanders-style/common';
+import { VlAlertClosedEvent } from './VlAlertClosedEvent';
```
we gaan typisch de files ook allemaal lower case houden; dus hier zou ik bv. eerder als filename vl-alert-closed-event.model.ts nemen

Goldflow reviewed (changes requested):
dag Wim

Changes lijken goed, behalve dan de filename van `VlAlertClosedEvent`, naar consistentie met de rest de filenames in ons project zou het goed zijn dat dit aangepast wordt.

Daarnaast, eventueel (maar enkel als je hier tijd voor vindt) zou je nog een test kunnen bijschrijven of bestaande test uitbreiden in: `apps/storybook-e2e/src/e2e/components/alert/vl-alert.stories.cy.ts`

Je toevoeging zal sowieso niets breken dus voor mij is dit niet ook niet cruciaal als je dit niet doet.

ksvlaanderen committed:
feat: UIG-2577 - nx common-storybook module

Nx module toegevoegd voor alle gedeelde Storybook code
